pkgs/by-name/n2/n26/package.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule {
+  pname = "n26";
+  version = "unstable-2025-04-24";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#versioning
```suggestion
  version = "0-unstable-2025-04-24";
```